modules/Hoot/ui/sidebar/advancedOpts.js

```diff
@@ -414,7 +422,7 @@ export default class AdvancedOpts {
         notNumber.dispatch( isNumber ? 'mouseleave' : 'mouseenter' );
     }
 
-    createGroups(advOpts) {
+    createGroups(advOpts, hiddenOpts ) {
```

Comment:
Also here, as it pertains to the null check on 493.
Having a null check is ok, but I think if we can avoid null checks, that's desirable. 
To do that you could just make hiddenOpts parameter default to an empty list in the method definition.

So you would change `createGroups(advOpts, hiddenOpts ) {`  to `createGroups(advOpts, hiddenOpts = []) {`